src/overview/bare-metal.md

```diff
@@ -1,59 +1,57 @@
+| ESP32-C2 |           ✅           |                  ✅                   |                 ✅                 |               ✅               |
+| ESP32-C3 |           ✅           |                  ✅                   |                 ✅                 |               ✅               |
+| ESP32-C6 |           ✅           |                  ✅                   |                 ✅                 |               ✅               |
+| ESP32-S2 |           ✅           |                  ✅                   |                 ✅                 |               ✅               |
+| ESP32-S3 |           ✅           |                  ✅                   |                 ✅                 |               ✅               |
+| ESP32-H2 |           ⏳           |                  ⏳                   |                 ⏳                 |               ⏳               |
 
-Chip support for `no_std` requires LLVM/Clang support just like for `std`. However, this has no dependency on `esp-idf`. In addition to compiler support, it's necessary to have peripheral access crates (PAC) and hardware abstraction layers (HAL) for your desired chip.
+> **Note**:
+>
+> - ✅ in Wi-Fi/BLE/ESP-NOW means that the target supports, at least, one the listed technologies. For details see [Current support][esp-wifi-current-support] table of the esp-wifi repository.
+> - [ESP8266 HAL][esp-rs/esp8266-hal] is in maintenance mode and no further development will be done for this chip.
 
-Refer to the table below to see if your chip is supported. Please note that the `no_std` HALs are still in the early phases of development, so not all peripherals have had drivers implemented.
+[esp-wifi-current-support]: https://github.com/esp-rs/esp-wifi#current-support
+### Relevant `esp-rs` crates
 
-|   Chip   |    PAC    |    HAL    |
-| :------: | :-------: | :-------: |
-|  ESP32   |     ✅     |     ✅     |
-| ESP32-C2 |     ✅     |     ✅     |
-| ESP32-C3 |     ✅     |     ✅     |
-| ESP32-C6 | _planned_ | _planned_ |
-| ESP32-S2 |     ✅     |     ✅     |
-| ESP32-S3 |     ✅     |     ✅     |
-| ESP32-H2 | _planned_ | _planned_ |
-| ESP8266  |     ✅     |     ✅     |
+| Repository             | Description                                                |
+| ---------------------- | ---------------------------------------------------------- |
+| [esp-rs/esp-hal]       | Hardware abstraction layer                                 |
+| [esp-rs/esp-pacs]      | Peripheral access crates                                   |
+| [esp-rs/esp-wifi]      | Wi-Fi, BLE and ESP-NOW support                             |
+| [esp-rs/esp-alloc]     | Simple heap allocator                                      |
+| [esp-rs/esp-println]   | `print!`,  `println!`                                      |
+| [esp-rs/esp-backtrace] | Exception and panic handlers                               |
+| [esp-rs/esp-storage]   | Embedded-storage traits to access unencrypted flash memory |
```

Comment:
```suggestion
| [esp-rs/esp-storage]   | Embedded-storage traits to access unencrypted flash memory |

### When you might want to use bare metal (`no_std`)

- Small memory footprint: If your embedded system has limited resources and needs to have a small memory footprint, you will likely want to use bare-metal because `std` features add a significant amount of final binary size and compilation time.
- Direct hardware control: If your embedded system requires more direct control over the hardware, such as low-level device drivers or access to specialized hardware features you will likely want to use bare-metal because `std` adds abstractions that can make it harder to interact directly with the hardware.
- Real-time constraints or time-critical applications: If your embedded system requires real-time performance or low-latency response times because `std` can introduce unpredictable delays and overhead that can affect real-time performance.
- Custom requirements: bare-metal allows more customization and fine-grained control over the behavior of an application, which can be useful in specialized or non-standard environments.
```

I think adding this section from @JurajSadel [blogpost](https://blog.espressif.com/rust-embedded-a-development-power-duo-db43dae21206) can be really useful